### DISCUSSION TITLE
Adding a way to reference a specific comment rather than a lign of code.

### DISCUSSION THREAD

**Discussion by Max-Martin-Sherie** [LefterisXris/CodeTour#96] [Ideas]
I want to reference a function in my code but every time dome new code gets added I have to set it again to the new line number. I would like there to be a way to reference a line and have it dynamically adjust to where it is (maybe by adding a literal string comment reference system ?)

**Comment by LefterisXris** [comment]
Hi @Max-Martin-Sherie, thanks for your comment.
What you want, is really valid. One of the major limitations that the current version of CodeTour has, is exactly this. It references line numbers and not the structure itself (i.e. a method reference) and therefore it cannot adapt to code changes.
I have had this feature in my mind since the beginning of this tool (you can check issue https://github.com/LefterisXris/CodeTour/issues/23 for *Dynamic Update*). 
I'll try to find some time to provide this feature because I understand it will be very useful (it's also challenging though :) )

**Reply by Max-Martin-Sherie** [reply]
Maybe a simple way to begin would be by using a string litteral

**Reply by LefterisXris** [reply]
Using a string literal is just a workaround. IntelliJ provides support for method references by traversing the tree of the parsed file (internally). That's the correct way to do it and I would definitely prefer it.

Stay tuned for updates. I'll look at it soon.
